src/hooks/useWindowSize.js

```diff
@@ -5,19 +5,26 @@ function useWindowSize() {
     const { innerHeight, innerWidth } = w;
     const [width, setWindowWidth] = useState(innerWidth);
     const [height, setWindowHeight] = useState(innerHeight);
+    let timeoutId;
 
     const handleResize = () => {
         const { innerHeight, innerWidth } = w;
         setWindowWidth(innerWidth);
         setWindowHeight(innerHeight);
     };
 
+    const debounceResize = () => {
```

Comment:
you repeat the same function 2 times, you can upgrade the current hook or create a new one